pkg/kubectl/cmd/run.go

```diff
@@ -88,7 +88,7 @@ var (
 
 func NewCmdRun(f cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "run NAME --image=image [--env=\"key=value\"] [--port=port] [--replicas=replicas] [--dry-run=bool] [--overrides=inline-json] [--command] -- [COMMAND] [args...]",
+		Use:     `run NAME --image=img1 [--env="key=value"] [--port=port] [--replicas=replicas] [--dry-run=bool] [--overrides=inline-json] [--command] -- [COMMAND] [args...]`,
```

Comment:
why change `image` to `img1`, how about `image_name` ?  also I'm not sure should we allow to change here to break the compatibility, even it's a help message.